### PR TITLE
Email templates

### LINKED ISSUE #26: Improve the content of the auth emails
All emails related with authentication need to be reviewed and improved.
- Change the looks
- Change the content of messages

### PR DESCRIPTION
**Pull request by dethos**
This PR tries to improve the text of several email templates of the project ( close #26 )

### CODE REVIEW

File: hawkpost/templates/account/email/email_confirmation_subject.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
 {% autoescape off %}
-{% blocktrans %}Please Confirm Your E-mail Address{% endblocktrans %}
+{% blocktrans %}Hawkpost: Please Confirm Your E-mail Address{% endblocktrans %}
```
Comment (ricardopereira) [inline hawkpost/templates/account/email/email_confirmation_subject.txt]: Use `{{ site_name }}`?

File: hawkpost/templates/account/email/password_reset_key_subject.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
 {% autoescape off %}
-{% blocktrans %}Password Reset E-mail{% endblocktrans %}
-{% endautoescape %}
\ No newline at end of file
+{% blocktrans %}Hawkpost: Password Reset{% endblocktrans %}
```
Comment (ricardopereira) [inline hawkpost/templates/account/email/password_reset_key_subject.txt]: Use `{{ site_name }}`?

**Comment by ricardopereira** [comment]
Small comments but the overall is 👍

**Comment by dethos** [comment]
@ricardopereira fixed

**Comment by ricardopereira** [comment]
LGTM

**Pull request merged**